src/fsm.rs

```diff
@@ -69,22 +58,15 @@ impl<AF: AddressFamily> FSM<AF> {
     }
 
     fn recv_packets(&mut self, cx: &mut Context) -> io::Result<()> {
-        let mut buf = [0u8; 4096];
+        let mut recv_buf = [0u8; 4096];
+        let mut buf = tokio::io::ReadBuf::new(&mut recv_buf);
         loop {
-            let (bytes, addr) = match self.socket.poll_recv_from(cx, &mut buf) {
-                Poll::Ready(Ok((bytes, addr))) => (bytes, addr),
+            let addr = match self.socket.poll_recv_from(cx, &mut buf) {
+                Poll::Ready(Ok(addr)) => addr,
                 Poll::Ready(Err(err)) => return Err(err),
                 Poll::Pending => break,
             };
-            // Is moot for certain platforms (Windows will throw a <10040> error from poll_recv)
```

Comment:
I am not sure about this, but it seems that there is no way to do this is Tokio 0.3?

Reply:
Yeah looks like this was probably cleared up in Tokio - which is nice.